### ISSUE TITLE
Improve accordion section accessibility

### ISSUE THREAD

**Issue by fdalcin** [pressbooks/pressbooks-book-directory-fe#357]
This is related to #106 

- [x] Replace wrapping `article` tag with `div`
- [x] Add `aria-expanded` attribute to identify when content is collapsed or expanded
- [x] Include and exclude buttons should have `aria-pressed` attribute to identify when filter is applied or not
- [x] Include and exclude filters should have visually hidden content to identify what the button does (e.g. "include education")
- [x] Numeric filters should have visually hidden labels that help identify what is being applied (e.g. "minimum word count value")
- [x] "Go" buttons should have visually hidden content to inform what action is being executed
- [x] Searchable filters text inputs should be replaced by search inputs
- [x] Show more buttons should have visually hidden content to inform the user (e.g. "Show more from language. 2 items left")
- [x] Show less buttons should have visually hidden content to inform the user (e.g. "Show less from language")
- [x] Clear filter should have visually hidden content to inform the user (e.g. "Clear license filter")
